packages/commonwealth/server/routes/linkExistingAddressToChain.ts

```diff
@@ -164,13 +166,24 @@ const linkExistingAddressToChain = async (
     const role = await findOneRole(
       models,
       { where: { address_id: addressId } },
-      req.body.chain
+      req.body.chain,
     );
 
     if (!role) {
       await createRole(models, addressId, req.body.chain, 'member');
     }
 
+    const serverAnalyticsController = new ServerAnalyticsController();
+    serverAnalyticsController.track(
+      {
+        isCustomDomain: null,
```

Comment:
Seems suspicious. Should be req.get('host') !== SERVER_URL I think.

Seems to be the same everywhere in the app. We should probably refactor them to be correct.

Reply:
Or better yet, it could be added to body of the track method, then we could just remove this isCustomDomain from all the track calls options arguments.

Reply:
@kurtisassad 
Pardon me; I should have been more specific in my response. Your solution is very close, but needs a little bit of string wrangling. `req.get` returns just the domain, but (see `server/config.ts`) SERVER_URL is set to 'https`/`http` plus the domain.